src/performance.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Performance monitor, holds reference to PerformanceMark instances.
+ */
+export class PerformanceMonitor {
+    constructor () {
+        this.marks = []
+    }
+
+    /**
+     * Create performance marker
+     * @param {string} name
+     * @returns {PerformanceMark}
+     */
+    mark (name) {
+        const mark = new PerformanceMark(name)
+        this.marks.push(mark)
+        return mark
+    }
+
+    /**
+     * Measure all performance markers
+     */
+    measure () {
```

Comment:
```suggestion
    measureAll () {
```

Reply:
NAB: because you can `measure` a `Mark`, how about making this `measureAll` for call-site clarity?

Reply:
Renamed :)